### PR TITLE
Rework code so robot frame is base_link is the center of the robot instead of the zed

### PR DESCRIPTION
**Pull request by jmblixt3**

### CODE REVIEW

File: src/isaac_ros/isaac_ros_launch/launch/nav2_gazebo_launch.py [hunk 1]
```diff
@@ -108,6 +108,14 @@ def generate_launch_description():
         name="frame_id_renamer",
     )
 
+    #map transform
+    map_transform = Node(
```
Comment (Isopod00) [inline src/isaac_ros/isaac_ros_launch/launch/nav2_gazebo_launch.py]: please remove everything in this PR related to map_transform

File: src/robot_description/models/master_ASM/urdf/master_ASM.urdf [hunk 2]
```diff
@@ -140,14 +141,6 @@ Stephen Brawner (brawner@gmail.com)
       <topic>/depth/image</topic>
       <info_topic>/depth/camera_info</info_topic>
     </sensor>
-    <sensor name="imu_sensor" type="imu">
```
Comment (Isopod00) [inline src/robot_description/models/master_ASM/urdf/master_ASM.urdf:123]: if you are going to remove the imu sensor, then remove it from the ros gz bridge too please

**Review by Isopod00 — CHANGES_REQUESTED** [review]

File: src/isaac_ros/isaac_ros_launch/isaac_ros_launch/odom_publisher.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        odom_to_base_tf.header.stamp = self.get_clock().now().to_msg()
+        odom_to_base_tf.header.frame_id = "odom"
+        odom_to_base_tf.child_frame_id = "base_link"
+
+        # since the odom frame is not the same as the camera frame, we need to get the transform between the two then apply it to the odom -> base_link transform
+        base_to_camera_tf: TransformStamped = self.tf_buffer.lookup_transform(
+            "base_link", "zed2i_camera_link", odom_to_base_tf.header.stamp
+        )
+
+        odom_to_base_tf.transform.translation.x = (
```
Comment (jmblixt3) [inline src/isaac_ros/isaac_ros_launch/isaac_ros_launch/odom_publisher.py:48]: Is this correct

File: src/robot_description/models/master_ASM/urdf/master_ASM.urdf.xacro [hunk 4]
```diff
@@ -49,9 +49,9 @@ Stephen Brawner (brawner@gmail.com)
   <xacro:zed_camera name="zed2i" model="zed2i">
   </xacro:zed_camera>
   <joint name="camera_joint" type="fixed">
-    <child link="base_link"/>
-    <parent link="zed2i_camera_link"/>
-    <origin xyz="-0.343 -0.009 -0.1254" rpy="0 -0.52359878 0" />
```
Comment (Isopod00) [inline src/robot_description/models/master_ASM/urdf/master_ASM.urdf.xacro:54]: why was the rotation on this deleted? is the zed not angled down anymore now?

File: config/nvblox/specializations/nvblox_zed.yaml [hunk 5]
```diff
@@ -1,9 +1,6 @@
 /**:
   ros__parameters:
     # esdf settings
-    esdf_slice_height: -0.3
-    esdf_2d_min_height: 0.0
```
Comment (Isopod00) [inline config/nvblox/specializations/nvblox_zed.yaml:5]: why are these lines deleted?

File: config/nvblox/specializations/nvblox_isaac_sim.yaml [hunk 6]
```diff
@@ -1,3 +0,0 @@
-/**:
-  ros__parameters:
```
Comment (Isopod00) [inline config/nvblox/specializations/nvblox_isaac_sim.yaml:2]: why are these lines deleted? did you mean to delete the whole file instead?

Comment (jmblixt3) [inline config/nvblox/specializations/nvblox_isaac_sim.yaml:2]: I did

File: config/nvblox/specializations/nvblox_zed.yaml (hunk 5 above)
Comment (jmblixt3) [reply config/nvblox/specializations/nvblox_zed.yaml:5]: They are duplicated in this file and in zed_common, so I kept the ones in zed common

File: config/nvblox/specializations/nvblox_isaac_sim.yaml (hunk 6 above)
Comment (Isopod00) [inline config/nvblox/specializations/nvblox_isaac_sim.yaml:2]: Why is this file deleted? Because it is still referenced in nvblox.launch.py, so I don't think it should be deleted.

Comment (jmblixt3) [reply config/nvblox/specializations/nvblox_isaac_sim.yaml:2]: The referenced code was deleted

File: config/nvblox/specializations/nvblox_zed.yaml (hunk 5 above)
Comment (jmblixt3) [reply config/nvblox/specializations/nvblox_zed.yaml:5]: nvblox_base.yaml*

**Review by Isopod00 — APPROVED** [review]
